settings.py

```diff
@@ -29,4 +38,36 @@
 SAVE_IN_FILE = False
 
 # Refresh assets from the error file
-REFRESH_FROM_FILE = False
\ No newline at end of file
+REFRESH_FROM_FILE = False
+
+# Automatically get the good url for the selected chain
```

Comment:
Can we move this logic from the settings file to somewhere else? I just don't feel it is necessary to have it here, since the goal is the settings file is to have one small file where all the important configurations can be made. Also, if you want, we can name it config.py, up to you.

Reply:
You're totally right, it will for sure be better at the beginning of the `main.py` file.
I'll fix that quickly 😁